test/ctia/test_helpers/aggregate.clj

```diff
@@ -50,17 +50,29 @@
                     (vector? (first values)) flatten-list-values)]
     (map string/lower-case flattened)))
 
+(defn- check-from-to
+  [from to]
+  (is (<= (time/in-years
+           (time/interval (tc/from-string from)
+                          (tc/from-string to)))
+          1)
+      "[from to[ should not exceed one year"))
+
 (defn- test-cardinality
   "test one field cardinality, examples are already created."
   [examples entity field]
   (testing (format "cardinality %s %s" entity field)
     (let [expected (-> (normalized-values examples field)
                        set
-                       count)]
-      (is (= expected (cardinality entity
-                                   {:query "*"
-                                    :from "2020-01-01"}
-                                   {:aggregate-on (name field)}))))))
+                       count)
```

Comment:
I'd assert that `count` is `pos?` (since probably that's always the case)

---

Similar suggestions for the rest of this ns